dsl/jpql/src/main/kotlin/com/linecorp/kotlinjdsl/dsl/jpql/Jpql.kt

```diff
@@ -536,6 +536,56 @@ open class Jpql : JpqlDsl {
         return Expressions.div(this.toExpression(), value.toExpression())
     }
 
+    /**
+     * Creates an expression that represents the mod of values.
+     * The values are each enclosed in parentheses.
+     *
+     * This is the same as ```(value1) % (value2)```.
+     */
+    @JvmName("modWithParentheses")
+    @SinceJdsl("3.3.2")
```

Comment:
Please change the base branch from main to develop and specify the version as 3.4.0. 🙏